### ISSUE TITLE
What do these two paths mean?（--metaA_path  --metaB_path）

### ISSUE THREAD

**Issue by Zhaoyu321** [mit-han-lab/gan-compression#102]
Hello! How to get the files under these two paths (--metaA_path  --metaB_path) when I train a supernet. In other words, how do I get the files under these two paths by training on my own dataset.
Thanks!

**Comment by lmxyy** [comment]
They are just the metafiles that define the dataset splits. For example, `train1` includes all the images in the training set, while  `train2` includes all the images in the validation set (or subtrain set) for the evolution search. `train1+train2` is used for fine-tuning.

**Comment by lmxyy** [comment]
You could also split the training set to `train1` and `train2` yourself.

**Comment by Zhaoyu321** [comment]
I understand, thank you very much for your reply!
